src/main.cpp

```diff
@@ -609,6 +626,8 @@ int main ( int argc, char** argv )
                 CInstPictures::UpdateTableOnLanguageChange();
             }
 
+            CClientRpc ClientRpc ( &Client, &RpcServer );
```

Comment:
Same as above - only do this if requested.

Reply:
Done — CClientRpc is no longer allocated without a CRpcServer.

bbb062387bf35e1f755837329324bb168ae643c8